src/EnergyPlus/HeatRecovery.cc

```diff
@@ -2138,7 +2148,7 @@ namespace HeatRecovery {
             //   (supply air stream bypass mass flow rate proportional to ControlFraction except when frost control is active)
             if (this->ControlToTemperatureSetPoint) {
                 if ((this->SupInTemp - this->SupOutTemp) != 0.0) {
-                    if ((this->SupInTemp < HXTempSetPoint && this->SupOutTemp > HXTempSetPoint) ||
+                    if ((this->SupInTemp<HXTempSetPoint &&this->SupOutTemp> HXTempSetPoint) ||
```

Comment:
Clang is acting wierd here. I manually fixed this and Clang set it back this way.

Reply:
I think it's a difference between the Clang-format version that CI uses and that you probably have locally.  It is getting very confused about the less than and greater than brackets, and thinking that they are actually template argument delimiters.  We can work around it and get rid of the issue by adding an extra set of parentheses around the less/greater than checks, or we can create a couple intermediate variables for the comparisons.  Anyway, no biggy, I can fix it here and push up the change.